nncf/torch/dynamic_graph/patch_pytorch.py

```diff
@@ -243,6 +243,35 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def get_disable_patching_wrapper(f):
+    """
+    :param: f: A callable object to be wrapped.
+    :return: A wrapper of a function that unpatches torch operators before the
+             function call and patches them back after the call.
+    """
+
+    @functools.wraps(f)
+    def wrapper(*args, **kwargs):
+        with disable_patching():
+            return f(*args, **kwargs)
+
+    return wrapper
+
+
+def module_call_wrapper(module_call):
+    wrapped_module_call = wrap_module_call(module_call)
+    unpatched_module_call = get_disable_patching_wrapper(module_call)
+
+    @functools.wraps(module_call)
+    def wrapper(self, *args, **kwargs):
+        # Check if model was patched by torch dynamo during compilation
+        if "_torchdynamo_orig_callable" in self.forward.__dict__:
+            unpatched_module_call(self, *args, **kwargs)
```

Comment:
Is it safe to use this condition here? I mean is it likely for torch dynamo to change the `_torchdynamo_orig_callable` attribute?

Reply:
Much better, thanks 👍